src/core/core.scale.js

```diff
@@ -669,7 +669,7 @@ class Scale extends Element {
 		var scaleLabelOpts = opts.scaleLabel;
 		var gridLineOpts = opts.gridLines;
 		var display = me._isVisible();
-		var isBottom = opts.position === 'bottom';
+		var isBottom = opts.position === 'bottom' || (opts.position === 'center' && opts.axis === 'x');
```

Comment:
It bothers me a bit that its considered to be bottom, when its center. Could rename the variable to something more accurate.

Reply:
I think the logic was to know when labels are below the ticks, so I've renamed the variable